File: Daily Test/C/Reverse Odd Row 8.4.2026.c
#include <stdio.h>

int main() {
    int n;
    scanf("%d", &n);

    int mat[n][n];

    for(int i = 0; i < n; i++)
        for(int j = 0; j < n; j++)
            scanf("%d", &mat[i][j]);

    for(int i = 0; i < n; i++){
        for(int j = 0; j < n; j++){
            if(i % 2 == 0)
                printf("%d ", mat[i][n-1-j]);
            else
                printf("%d ", mat[i][j]);
        }
        printf("\n");
    }

    return 0;
}

File: Daily Test/Java/Reverse Odd Row 8.4.2026.java
import java.util.*;

public class Main {
    public static void main(String[] args) {
        Scanner s = new Scanner(System.in);
        int n = s.nextInt();
        int[][] mat = new int[n][n];

        for(int i=0;i<n;i++)
            for(int j=0;j<n;j++)
                mat[i][j]=s.nextInt();

        for(int i=0;i<n;i++){
            for(int j=0;j<n;j++){
                if(i%2==0)
                    System.out.print(mat[i][n-1-j]+" ");
                else
                    System.out.print(mat[i][j]+" ");
            }
            System.out.println();
        }
    }
}